### DISCUSSION TITLE
Want to change USER_ID and add folder to an existing Crashplan config

### DISCUSSION THREAD

**Discussion by septeven** [jlesage/docker-crashplan-pro#435] [General]
Hi,
I've been using Crashplan through your docker image for several years now, and it works great. Thank you for the great work btw!

At that time, I did set up the container using these commands : 
```
	docker run -d \
	--name=crashplan-pro \
	-e USER_ID=0 -e GROUP_ID=0 \
	-e CRASHPLAN_SRV_MAX_MEM=1600M \
	-p 5800:5800 \
	-v /volume1/docker/appdata/crashplan-pro:/config:rw \ 
	-v /volume1/:/volume1:ro \
	-- restart always \
	jlesage/crashplan-pro
```
I know that mapping have changed with DSM7 (-v /volume1/:/volume1:ro \ not allowed anymore), and I now have to map every each folder under volume1. 

My question is, now that I have a bit more of knowledge regarding docker, synology, etc... I'd like to create a specific user for crashplan (I know how to create one, and how to use the -e USER_ID, GROUP_ID for declaring it) and I'd like to add new folders to my backup as well.

As I don't want to screw up anything with my backup, I would like to know - how should I proceed to make those changes ?

**Comment by jlesage** [comment]
- When you change the user (via USER_ID, GROUP_ID), make sure it has read access to the files you want to backup.
- Splitting `-v /volume1/:/volume1:ro` into multiple mappings should also not cause any problem.  Just make sure that the content of `/volume1` inside the container remain the same at the end.
- Adding new folders to the backup should be as simple as adding a new mappings (e.g. `-v /volume1/NewFolder:/volume1/NewFolder:ro`).
- Obviously, make sure to keep your `/config` the same.
